examples/mautic-example-nginx-ssl/README.md

```diff
@@ -3,16 +3,17 @@ This example shows how to configure nginx to act as a reverse proxy with ssl sup
 #### What is handled in this docker-compose.yml:
 
 * Setup of mysql and mautic containers
-* Setup of a nginx container with custom vhost configuration (nginx.conf)
-* Automatic creation of self-signed certificate
+* Setup of mysql db on managed or remote server
+* Setup of a nginx container with custom vhost configuration (nginx.conf)and secure with letsencrypt certs
 
 #### How to use this example:
 
-1. run ```docker-compose up``` in this directory
-2. add this line to your /etc/hosts file ```127.0.4.123 mautic.local```
-3. access https://mautic.local
-4. add the presented certificate to a trusted certificates list in your browser (the certificate is a self-signed certificate created on the first run of this example)
-5. go through Mautic setup (fill ```mauticdbpass``` as mysql password on db setup page
+1. run ```docker-compose -f docker-compose-managed_database.yaml up``` in this directory
+2. add these lines to your /etc/hosts file 
+```127.0.0.1       pilotdev9.eastbayinfo.org ```
+``` 127.0.4.123 mautic.local ```
+3. access https://pilotdev9.eastbayinfo.org
+4. go through Mautic setup (fill ```mauticdbpass``` as mysql password on db setup page
 6. test Mautic
```

Comment:
```suggestion
6. Test Mautic
```

Please also correct the typos/text in the following section from the other file where I have made suggestions!